repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
```

Comment:
Ah.  I see.  I had not tried to click on one of the TOC links.  I assumed it would work like a regular TOC link instead of being fancy and using the link we put in the section header.

So...  have you already looked into specifying CSS for a specific page by chance?  I suppose worst case scenario we could add an HTML check and include the CSS file for the Filters page, but I would hope there's an easier/cleaner way.